DiseasesExpanded/Cures/RadShotConfig.cs

```diff
@@ -39,11 +39,12 @@ private void DefineRecipe()
             };
             recipe = new ComplexRecipe(ComplexRecipeManager.MakeRecipeID(ApothecaryConfig.ID, (IList<ComplexRecipe.RecipeElement>)ingredients, (IList<ComplexRecipe.RecipeElement>)results), ingredients, results)
             {
-                time = 100f,
+                time = 200f,
```

Comment:
why the time is doubled?

Reply:
The cure is considered advanced so I took the same time from Serum Vial (AdvancedCureConfig).